src/util/util/cdn.ts

```diff
@@ -75,7 +75,7 @@ export async function handleFile(
 
 export async function deleteFile(path: string) {
 	const response = await fetch(
-		`${Config.get().cdn.endpointPrivate || "http://localhost:3001"}${path}`,
+		`${Config.get().cdn.endpointPrivate || "http://127.0.0.1:3001"}${path}`,
```

Comment:
will break ipv6 support
fosscord, unlike discord.com, supports ipv6

Reply:
Thanks @erkinalp, completely forgot about IPv6, https://en.wikipedia.org/wiki/Localhost and others say it can be both 6 & 4, so there shouldn't be a need to detect.